src/client.ts

```diff
@@ -105,6 +106,11 @@ export class SQLocal {
 			| GetInfoMessage
 		>
 	): Promise<OutputMessage> => {
+		if (!this.worker) {
+			throw new Error(
+				'This SQLocal client does not have a worker. This is likely due to SSR or worker failing to initialize.'
```

Comment:
Let's make the message: "This SQLocal client is not connected to a database. This is likely due to the client being initialized in a server-side environment."

Let's also add an empty line after this `if` statement.